From 87775b254aee1ed17bf2dddf2ba3e7df17b1add6 Mon Sep 17 00:00:00 2001
From: Undef <gitlab@undef.tools>
Date: Sun, 27 Aug 2023 22:16:50 +0200
Subject: [PATCH] Minimize width of left pane on open

---
 ts/state/selectors/items.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ts/state/selectors/items.ts b/ts/state/selectors/items.ts
index 75ebfb353..ee2117f0a 100644
--- a/ts/state/selectors/items.ts
+++ b/ts/state/selectors/items.ts
@@ -23,7 +23,7 @@ import { generateUsernameLink } from '../../util/sgnlHref';
 import * as Bytes from '../../Bytes';
 import { getUserNumber, getUserACI } from './user';
 
-const DEFAULT_PREFERRED_LEFT_PANE_WIDTH = 320;
+const DEFAULT_PREFERRED_LEFT_PANE_WIDTH = 109;
 
 export const getItems = (state: StateType): ItemsStateType => state.items;
 
-- 
2.42.0

