From a8a789c22092d8a56dfb94af64a3cd073c9e5c3f Mon Sep 17 00:00:00 2001
From: Undef <gitlab@undef.tools>
Date: Sat, 10 Dec 2022 12:20:46 +1100
Subject: [PATCH] Minimize width of left pane on open

---
 ts/state/selectors/items.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ts/state/selectors/items.ts b/ts/state/selectors/items.ts
index 8fbe2e6b3..f24618c4b 100644
--- a/ts/state/selectors/items.ts
+++ b/ts/state/selectors/items.ts
@@ -20,7 +20,7 @@ import { isBeta } from '../../util/version';
 import { DurationInSeconds } from '../../util/durations';
 import { getUserNumber, getUserACI } from './user';
 
-const DEFAULT_PREFERRED_LEFT_PANE_WIDTH = 320;
+const DEFAULT_PREFERRED_LEFT_PANE_WIDTH = 109;
 
 export const getItems = (state: StateType): ItemsStateType => state.items;
 
-- 
2.35.1

