From da525dd2a3d8c49697653372c96f33129b5974ae Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Mon, 18 Jul 2022 22:56:13 +0000
Subject: [PATCH] reduce minimum dimensions to allow for mobile screens

---
 app/main.ts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/app/main.ts b/app/main.ts
index 737ce97e6..a4eff1d2c 100644
--- a/app/main.ts
+++ b/app/main.ts
@@ -550,14 +550,14 @@ function handleCommonWindowEvents(
   }
 }
 
-const DEFAULT_WIDTH = 800;
+const DEFAULT_WIDTH = 360;
 const DEFAULT_HEIGHT = 610;
 // LARGEST_LEFT_PANE_WIDTH = 380
 // TIMELINE_WIDTH = 300
 // TIMELINE_MARGIN = 16 + 16
 // 712 = LARGEST_LEFT_PANE_WIDTH + TIMELINE_WIDTH + TIMELINE_MARGIN
-const MIN_WIDTH = 712;
-const MIN_HEIGHT = 550;
+const MIN_WIDTH = 360;
+const MIN_HEIGHT = 280;
 const BOUNDS_BUFFER = 100;
 
 type BoundsType = {
-- 
2.35.1
