From f9c5bc0d6d9333ba2fcc206114d66985f237d1d0 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Thu, 7 Sep 2023 22:01:58 +0200
Subject: [PATCH] Remove Stories from one file

---
 ts/components/NavTabs.tsx | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/ts/components/NavTabs.tsx b/ts/components/NavTabs.tsx
index aeae1e0fa..5143f3940 100644
--- a/ts/components/NavTabs.tsx
+++ b/ts/components/NavTabs.tsx
@@ -270,20 +270,6 @@ export function NavTabs({
               markedUnread: false,
             }}
           />
-          {storiesEnabled && (
-            <NavTabsItem
-              i18n={i18n}
-              id={NavTab.Stories}
-              label="Stories"
-              iconClassName="NavTabs__ItemIcon--Stories"
-              hasError={hasFailedStorySends}
-              unreadStats={{
-                unreadCount: unreadStoriesCount,
-                unreadMentionsCount: 0,
-                markedUnread: false,
-              }}
-            />
-          )}
         </TabList>
         <div className="NavTabs__Misc">
           <ContextMenu
-- 
2.42.0

