From da525dd2a3d8c49697653372c96f33129b5974ae Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Thu, 03 Nov 2022 00:00:00 +0000
Subject: [PATCH] remove stories from one file

---
 ts/components/MainHeader.tsx | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/ts/components/MainHeader.tsx b/ts/components/MainHeader.tsx
index a4e9749d1..1420746b8 100644
--- a/ts/components/MainHeader.tsx
+++ b/ts/components/MainHeader.tsx
@@ -229,21 +229,6 @@ export class MainHeader extends React.Component<PropsType, StateType> {
             : null}
         </Manager>
         <div className="module-main-header__icon-container">
-          {areStoriesEnabled && (
-            <button
-              aria-label={i18n('stories')}
-              className="module-main-header__stories-icon"
-              onClick={toggleStoriesView}
-              title={i18n('stories')}
-              type="button"
-            >
-              {unreadStoriesCount ? (
-                <span className="module-main-header__stories-badge">
-                  {unreadStoriesCount}
-                </span>
-              ) : undefined}
-            </button>
-          )}
           <button
             aria-label={i18n('newConversation')}
             className="module-main-header__compose-icon"
--
2.37.3
