From 173384c7310b8c8be801b5f29d51eb1a40d2b7f3 Mon Sep 17 00:00:00 2001
From: name <name@example.com>
Date: Fri, 15 Apr 2022 05:46:31 +0000
Subject: [PATCH] Build better-sqlite3 dynamically

---
 deps/sqlite3.gyp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/sqlite3.gyp b/deps/sqlite3.gyp
index 7774bcb..fa88222 100755
--- a/deps/sqlite3.gyp
+++ b/deps/sqlite3.gyp
@@ -99,7 +99,7 @@
           'link_settings': {
             'libraries': [
               # This statically links libcrypto, whereas -lcrypto would dynamically link it
-              '<(SHARED_INTERMEDIATE_DIR)/sqlite3/OpenSSL-linux-<(target_arch)/libcrypto.a'
+              '-lcrypto'
             ]
           }
         }],
-- 
2.35.1

