From 75a2014669d914a6502a82f8b4f1db01a0419b08 Mon Sep 17 00:00:00 2001
From: undef <undef>
Date: Sun, 17 Oct 2021 04:36:54 +0000
Subject: [PATCH] Work around rust compiler bug on Arm64.

---
 rust-toolchain.toml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/rust-toolchain.toml b/rust-toolchain.toml
index 74e786c..b1485fd 100644
--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2021-09-19"
+channel = "nightly-2021-09-16"
 components = [
     "rust-src",
 ]
@@ -17,4 +17,5 @@ targets = [
 #    "x86_64-apple-ios-macabi",
     "x86_64-linux-android",
     "x86_64-unknown-linux-gnu",
+    "aarch64-unknown-linux-gnu",
 ]
-- 
2.33.0
